CHANGELOG.md

```diff
@@ -56,6 +56,10 @@ Version <dev> adds Apache Kafka instrumentation for the rdkafka and ruby-kafka g
 
 [PR#2851](https://github.com/newrelic/newrelic-ruby-agent/pull/2851)
 
+- **Feature: Collect just MIME type from AcionDispatch requests**
```

Comment:
Maybe at a byline to the summary for this version as well.

Reply:
Looking for version in title? https://github.com/newrelic/newrelic-ruby-agent/pull/2855/commits/60dcfc9872bbe815915f0af0cd75cbc0de8df625